youtube_dl/extractor/sbs.py

```diff
@@ -1,33 +1,66 @@
 # coding: utf-8
 from __future__ import unicode_literals
 
+import sys
+
 from .common import InfoExtractor
+from ..compat import (
+    compat_HTTPError,
+    compat_kwargs,
+    compat_str,
+)
 from ..utils import (
-    smuggle_url,
+    error_to_compat_str,
     ExtractorError,
+    float_or_none,
+    GeoRestrictedError,
+    int_or_none,
+    parse_duration,
+    parse_iso8601,
+    smuggle_url,
+    traverse_obj,
+    update_url_query,
+    url_or_none,
+    variadic,
 )
 
 
 class SBSIE(InfoExtractor):
     IE_DESC = 'sbs.com.au'
-    _VALID_URL = r'https?://(?:www\.)?sbs\.com\.au/(?:ondemand(?:/video/(?:single/)?|.*?\bplay=|/watch/)|news/(?:embeds/)?video/)(?P<id>[0-9]+)'
+    _VALID_URL = r'''(?x)
+        https?://(?:www\.)?sbs\.com\.au/(?:
+            ondemand(?:
+                /video/(?:single/)?|
+                /movie/[^/]+/|
+                /(?:tv|news)-series/(?:[^/]+/){3}|
+                .*?\bplay=|/watch/
+            )|news/(?:embeds/)?video/
+        )(?P<id>[0-9]+)'''
+    _EMBED_REGEX = [r'''(?x)]
+            (?:
+                <meta\s+property="og:video"\s+content=|
+                <iframe[^>]+?src=
+            )
+            (["\'])(?P<url>https?://(?:www\.)?sbs\.com\.au/ondemand/video/.+?)\1''']
 
     _TESTS = [{
-        # Original URL is handled by the generic IE which finds the iframe:
+        # Exceptional unrestricted show for testing, thanks SBS,
+        # from an iframe of this page, handled by the generic IE, now 404:
         # http://www.sbs.com.au/thefeed/blog/2014/08/21/dingo-conservation
```

Comment:
Is this where the url moved? https://www.sbs.com.au/programs/video/320403011771/Dingo-Conservation-The-Feed